narwhal/network/src/lib.rs

```diff
@@ -10,6 +10,7 @@
 #![allow(clippy::async_yields_async)]
 
 pub mod admin;
+pub mod anemo_ext;
```

Comment:
Do you think it's worth moving this to a shared location outside of narwhal, if we'd ever use it in sui code too?

Reply:
Yes i do! problem is that "shared" place has in the past been `mysten-network` and that's in another repo. So what we could probably do is once we move `mysten-network` into the sui repo then we can do some reshuffling.